lib/tmux_adapter.sh

```diff
@@ -1,7 +1,8 @@
 # Get the current path in the segment.
 get_tmux_cwd() {
-	local env_name=$(tmux display -p "TMUXPWD_#D" | tr -d %)
-	local env_val=$(tmux show-environment | grep --color=never "$env_name")
+	local env_name=$(tmux display -p "#D" | tr -d %)
+	# use pane_current_path, no need to set TMUXPWD
+	local env_val=$(tmux display-message -p -F "#{pane_current_path}" -t"$env_name")
```

Comment:
I tried this out in a tmux session, and it seems like -t to display-message do want a % before the name, otherwise I get e.g. "can't find pane 15".

If I remove the "tr -d %" it works for me. I'm using Tmux v2.1.